ci-operator/step-registry/openshift-extended/web-tests/openshift-extended-web-tests-ref.yaml

```diff
@@ -0,0 +1,13 @@
+ref:
+  as: openshift-extended-web-tests
+  from: tests-private-ui
+  commands: openshift-extended-web-tests-commands.sh
+  timeout: 1h0m0s
+  resources:
+    requests:
+      cpu: "2"
```

Comment:
@yapei better change to 1. now we change golang step to 1.

Reply:
@yapei how about this comment?

Reply:
@kuiwang02  no idea about what value it should be, based on our experience web tests normally need more CPU and Memory usage because browser consumes higher usage, HDYT?

Reply:
@yapei per comment ever coming from @liangxia (https://github.com/openshift/release/pull/28370), better change to 1 to make efficient. no worry because it is request, not limit.

Reply:
no problem, updated @kuiwang02